### PR TITLE
doc: add meeting minutes 2017-05-24

### PR DESCRIPTION
**Pull request by targos**

### CODE REVIEW

**Review by jasnell — APPROVED** [review]

File: meetings/2017-05-24.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## Q&A, Other
+
+
+## Upcoming Meetings
+ 
+* **Node.js Foundation Calendar**: https://nodejs.org/calendar
+ 
+Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.
+	
+* **TSC**: May 25, 20:00 UTC
```
Comment (mhdawson) [inline meetings/2017-05-24.md]: I think the calendar shows the next TSC meeting as 1 June as opposed to May 25

**Review by mhdawson — APPROVED** [review]
LGTM once date for next TSC meeting is fixed up.

File: meetings/2017-05-24.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Node Foundation CTC Meeting 2017-05-24
+## Links
+
+* **Audio recording**: <https://www.youtube.com/watch?v=jdi8QhexKQM>
```
Comment (refack) [inline meetings/2017-05-24.md]: This is the audio of last meeting.

Comment (refack) [reply meetings/2017-05-24.md]: Should be https://www.youtube.com/watch?v=JbC3TQ04G0U

Comment (refack) [reply meetings/2017-05-24.md]: Even better https://youtu.be/JbC3TQ04G0U?t=1206

**Pull request merged**